packages/gatsby-theme-carbon/src/components/NextPrevious/NextPrevious.js

```diff
@@ -150,6 +150,10 @@ const NextPreviousContainer = (props) => {
   };
 
   const getNextItem = () => {
+    if (!navigationList[navIndex]) {
```

Comment:
Is this still needed if you're getting the title from the filename if one doesn't exist?

Reply:
No, I'm surprised actually. This was a semi related challenge where the page wouldn't render if it wasn't in the nav list even if it had a title. (I often do this to dark launch pages)

Reverse code gremilins! 🚀